bin/lint-migrations-file/src/change_set/common.clj

```diff
@@ -15,7 +15,7 @@
 
 (s/def ::new-style-id
   (s/and string?
-         #(re-matches #"^v\d{2}\.\d{2}-\d{3}$" %)))
+         #(re-matches #"^v\d{2,}\.\d{2}-\d{3}$" %)))
```

Comment:
this should have been in the PR to switch to the new migration format but I missed this. Minor tweak. Allow >2 digit major versions. So when we get to 100.0 we don't have things start breaking